LaTeX/LaTeX.sublime-syntax

```diff
@@ -110,79 +110,89 @@ contexts:
 
   macros:
     - include: scope:text.tex#macros
-    - match: |-
-        (?x)
-        # The defining command
-        (
-          (\\)
-          (?:(?:new|renew|provide)command\*?)
-        )
-        \s*
-        # the command to be defined, possibly surrounded by braces
-        (?:
-          (\{)\s*(\\[A-Za-z@]+)\s*(\})
-          | (\\[A-Za-z@]+)
-        )
-        \s*
-        # optionally, the number of arguments. LaTeX allows only up to 9, so a single digit
-        (?:(\[)\s*(\d)\s*(\]))?
-        \s*
-        # optionally, a default argument value
-        (?:(\[)([^\]]*)(\]))?
-        \s*
-        # start of actual command definition
-        (\{)
+    - match: '(\\)(?:(?:new|renew|provide)command\*?)'
```

Comment:
No need for the double wrapping of no-capture group.

Reply:
I'm ambivalent about the trailing storage modifier. I know it was in the old one.